client/src/main/java/io/paradoxical/cassieq/api/client/CassandraQueueApi.java

```diff
@@ -56,5 +56,7 @@ static CassandraQueueApi createClient(String baseUri) {
             @Path("queueName") QueueName queueName,
             @Query("popReceipt") String popReceipt);
 
+    @DELETE("/api/v1/queues/{queueName}")
```

Comment:
all of our routes are doing this right now. should we make a task to not do this?